### PR TITLE
Website navigation

### PR DESCRIPTION
**Pull request by JoyTaribagshaw**
> ## Awesome Books
> ### The branch 'refactor-class' was created by @Obote  and @JoyTaribagshaw  following the requirements of the [Microverse Awesome books: a complete website with navigation](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website_v1_1.md).
> **In this branch we have converted the application into a fully functional website. We have added three new sections:**
> 
> * [Contact info section](https://github.com/Obote/AwesomeBooks/pull/4);
> * [Navigation bar](https://github.com/Obote/AwesomeBooks/pull/4);
> * [Footer](https://github.com/Obote/AwesomeBooks/pull/4).
> 
> ### General requirements:
> * there are no linter errors;
> * we used the correct GitHub Flow;
> * we documented our work in a professional way.
> * followed lists of best practices for JavaScript, HTML and CSS.
> 
> ### Functionality:
> * each link in the Navigation bar shows the corresponding section and hides the unnecessary sections.
> * when a section is visited the corresponding link in the Navigation bar gets highlighted.
> 
> ### Styling:
> We have added CSS styles to the application to make it match the wireframe.
> 

awesomebooks

### CODE REVIEW

**Comment by codecaiine** [comment]
Hi @JoyTaribagshaw @Obote  Can you go on slack ?

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @JoyTaribagshaw @Obote , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a descriptive summary . Please sure that your Pull Request has a short summary describing the introduced changes in more detail. 

pr


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by JoyTaribagshaw** [comment]
Thank you so much @codecaiine , I have made the required changes.
highlight

File: index.html [hunk 1]
```diff
@@ -6,18 +6,54 @@
...
+    <header>
+      <div class= "nav">
+     <a href="#" class="navbar-logo">Awesome Books</a>
+      <nav class="navbar">
+        <ul class="nav-list">
+          <li class="navbar-list bookList">List</li>
+          <li class="navbar-list form">Add new</li>
+          <li class="navbar-list contact">Contact</li>
+        </ul>
+       </nav>
```
Comment (Cabraham1) [inline index.html:20]: nice work so far 👍🏿  in your project but as per project requirements when any of the links are clicked, the main area changes to display only one of the three content sections. you can make use of clssList in css

File: javascript/index.js [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+//       } else {
+//         container.style.display = 'block';
+//       }
+//     });
+//   });
+// });
+// window.addEventListener('DOMContentLoaded', () => {
+//   hideAllContainers();
+//   containers[0].style.display = 'block';
+// });
```
Comment (Cabraham1) [inline javascript/index.js]: - [ ]  please always do well to remove commented code from your code base so as to not make your code messy and unprofessional 😉

File: javascript/index.js [hunk 3]
```diff
@@ -56,3 +89,30 @@ class BookStore {
...
+// console.log('show books');
+// });
+
+// navContact.addEventListener('click', () => {
+//   showContact.style.display = 'flex';
+//   showContact.style.display = 'none';
+//   showAddBook.style.display = 'none';
+//   showBookList.style.display = 'flex';
+// console.log('show contact');
+// });
```
Comment (Cabraham1) [inline javascript/index.js]: - [ ] please always do well to remove commented code from your code base so as to not make your code messy and unprofessional 😉

File: javascript/index.js [hunk 4]
```diff
@@ -56,3 +89,30 @@ class BookStore {
 
 // eslint-disable-next-line no-unused-vars
```
Comment (Cabraham1) [inline javascript/index.js]: - [ ] Please you are not allowed to disable any eslint in your project please do well to enable eslint and fix any error locally on your machine before making a PR

**Review by Cabraham1 — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @JoyTaribagshaw   :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Impressive UI
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers, and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (peterrobert) [reply index.html:20]: This change has been done well

File: javascript/index.js (hunk 2 above)
Comment (peterrobert) [reply javascript/index.js]: This change has been done well

File: javascript/index.js (hunk 3 above)
Comment (peterrobert) [reply javascript/index.js]: This change has been done well

File: javascript/index.js (hunk 4 above)
Comment (peterrobert) [reply javascript/index.js]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @JoyTaribagshaw, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**